doc/orchestrator.rst

```diff
@@ -165,3 +165,460 @@ A full example of configuring KeyDB to run in SmartSim is shown below
+workflow summary with ``Experiment.summary()``.
+
+.. code-block:: python
+
+  exp.stop(single_shard_db, multi_shard_db)
+  logger.info(exp.summary())
+
+When you run the experiment, the following output will appear.
+
+.. code-block:: bash
+
+  00:00:00 system.host.com SmartSim[#####]INFO
+  |    | Name                         | Entity-Type   | JobID       | RunID   | Time    | Status    | Returncode   |
+  |----|------------------------------|---------------|-------------|---------|---------|-----------|--------------|
+  | 1  | colo_model                   | Model         | 1538905.8   | 0       | 3.5465  | Completed | 0            |
+  | 2  | single_shard_db_identifier_0 | DBNode        | 1538905.5   | 0       | 73.1798 | Cancelled | 0            |
+  | 3  | multi-shard-db-identifier    | DBNode        | 1538905.6+2 | 0       | 49.8503 | Cancelled | 0            |
+
+Source Code
+-----------
+
+Experiment script
+^^^^^^^^^^^^^^^^^
+.. code-block:: python
+
+  import numpy as np
+  from smartredis import Client
+  from smartsim import Experiment
+  from smartsim.log import get_logger
+  import sys
+
+  exe_ex = sys.executable
+  logger = get_logger("Multidb Experiment Log")
+  exp = Experiment("getting-started-multidb", launcher="auto")
+
+  single_shard_db = exp.create_database(port=6379, db_nodes=1, interface="ib0", db_identifier="single_shard_db_identifier")
+  exp.generate(single_shard_db, overwrite=True)
+
+  multi_shard_db = exp.create_database(port=6380, db_nodes=3, interface="ib0", db_identifier="multi-shard-db-identifier")
+  exp.generate(multi_shard_db, overwrite=True)
+
+  exp.start(single_shard_db, multi_shard_db)
+
+  driver_client_single_shard = Client(cluster=False, address=single_shard_db.get_address()[0], logger_name="Single shard db logger")
+  driver_client_multi_shard = Client(cluster=True, address=multi_shard_db.get_address()[0], logger_name="Multi shard db logger")
+
+  array_1 = np.array([1, 2, 3, 4])
+  driver_client_single_shard.put_tensor("tensor_1", array_1)
+
+  array_2 = np.array([5, 6, 7, 8])
+  driver_client_multi_shard.put_tensor("tensor_2", array_2)
+
+  check_single_shard_db_tensor_incorrect = driver_client_single_shard.key_exists("tensor_2")
+  check_multi_shard_db_tensor_incorrect = driver_client_multi_shard.key_exists("tensor_1")
+  logger.info(f"The multi shard array key exists in the incorrect database: {check_single_shard_db_tensor_incorrect}")
+  logger.info(f"The single shard array key exists in the incorrect database: {check_multi_shard_db_tensor_incorrect}")
+
+  model_settings = exp.create_run_settings(exe=exe_ex, exe_args="/lus/scratch/richaama/model_ex.py")
+  model_settings.set_nodes(1)
+  model_settings.set_tasks_per_node(1)
+  model = exp.create_model("colo_model", model_settings)
+  model.colocate_db_tcp(db_identifier="colo_db_identifier")
+  exp.start(model, block=True, summary=True)
+
+  exp.stop(single_shard_db, multi_shard_db)
+  logger.info(exp.summary())
+
+Application Script
```

Comment:
maybe a good idea to add the name of the scripts that contain this information that you are using in your examples above (eg.  exe_args="/lus/scratch/richaama/model_ex.py") is the application script. Also probably a good idea to have step by step instructions that show where files should be located in relation to smartsim, the contents of those files, what should be run (eg. only running experiment script, which uses the application script).

Reply:
I think a step by step on what is needed to run the example would be useful (allocation, any module load stuff, the structure of the file system, contents of files, what exactly to run)

Reply:
I added this but let me know what u think

Reply:
Im not sure how specific I can be on the setup instructions bc people might be installing in many dif ways, but hmmm @ashao and @juliaputko let me know if I can put submitting an allocation or anything else? It should be the same on all slurm machines right? But someone might want to change the launcher? so I could just say make sure you have allocated resources, lemme know what yall think

Reply:
I'd say point them at the beginning of the section, you could just say something along the lines of "This section assumes that you have already installed and built SmartSim and SmartRedis. Please refer to Section "Installing SmartSim" <and include the link to that document> for further details. For simplicity, this example also assumes that you are running on your own local machine."

Reply:
dern!